### ISSUE TITLE
installation of Lyx failed because libhunspell is lacking (Bugzilla Bug 2519)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#2519]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 2519
Date: 2019-07-25 12:02:53 +0000
From: Nicolas TAULIER &lt;<nicolas.taulier@gmail.com>&gt;
To:   AngryPenguin &lt;<angrypenguinpoland@gmail.com>&gt;
CC:   angrypenguinpoland@gmail.com, @benbullard79, bugs@openmandriva.org, nicolas.taulier@gmail.com, nix.or.die@gmail.com, nobodydead@gmail.com, @tpgxyz

Last updated: 2019-07-29 22:46:56 +0000

## Comment 19692
Date: 2019-07-25 12:02:53 +0000
From: Nicolas TAULIER &lt;<nicolas.taulier@gmail.com>&gt;

After a fresh and updated installation of LX4, I tried to install lyx 2.3.2 rev1 from rock-x86_64, this installation failed with the error message :

Problem: conflicting requests
- nothing provides libhunspell-1.6.so.0()(64bit) needed by lyx-2.3.2-1.x86_64

Nicolas

## Comment 19695
Date: 2019-07-26 10:50:25 +0000
From: Nicolas TAULIER &lt;<nicolas.taulier@gmail.com>&gt;

A little precision, lib64hunspell1.7_0 has been installed by LX4 but lyx is looking for the 1.6 version.

Nicolas

## Comment 19696
Date: 2019-07-26 14:47:06 +0000
From: Nicolas TAULIER &lt;<nicolas.taulier@gmail.com>&gt;

I have installed lib64hunspell1.6 from the Main update repository of LX3.
Then I tried to install again  Lyx, now I have the error message :

package lyx-2.3.2-1.x86_64 requires texlive-scheme-full, but none of the providers can be installed
 - package texlive-scheme-full-20190228-1.noarch requires texlive-collection-genericextra, but none of the providers can be installed
 - conflicting requests
 - nothing provides texlive-c-pascal needed by texlive-collection-genericextra-1:20140621-5.noarch

So I will also open a new bug report for texlive-collection-genericextra ...

Nicolas

## Comment 19698
Date: 2019-07-26 15:39:48 +0000
From: Nicolas TAULIER &lt;<nicolas.taulier@gmail.com>&gt;

I tried to dig out further to see if I can make it work. So I decided to install texlive-c-pascal and then I got :

package lyx-2.3.2-1.x86_64 requires texlive-scheme-full, but none of the providers can be installed
 - package texlive-scheme-full-20190228-1.noarch requires texlive-collection-fontsextra, but none of the providers can be installed\n - conflicting requests
 - nothing provides texlive-xits needed by texlive-collection-fontsextra-1:20190228-1.noarch
 - nothing provides texlive-gnu-freefont needed by texlive-collection-fontsextra-1:20190228-1.noarch

then I have installed texlive-gnu-freefont installed, and I have this: 

package lyx-2.3.2-1.x86_64 requires texlive-scheme-full, but none of the providers can be installed
 - package texlive-scheme-full-20190228-1.noarch requires texlive-collection-fontsextra, but none of the providers can be installed
 - conflicting requests
 - nothing provides texlive-xits needed by texlive-collection-fontsextra-1:20190228-1.noarch

After installing texlive-xits I got the message :

package lyx-2.3.2-1.x86_64 requires texlive-scheme-full, but none of the providers can be installed
 - package texlive-scheme-full-20190228-1.noarch requires texlive-collection-binextra, but none of the providers can be installed
 - conflicting requests
 - nothing provides texlive-texliveonfly needed by texlive-collection-binextra-1:20190311-1.noarch

this is the message I have in the bug #2520.  If I install texlive-texliveonfly I habe now another error message :

Problème: package lyx-2.3.2-1.x86_64 requires texlive-scheme-full, but none of the providers can be installed
 - package texlive-scheme-full-20190228-1.noarch requires texlive-collection-bibtexextra, but none of the providers can be installed
 - conflicting requests
 - nothing provides texlive-rsc needed by texlive-collection-bibtexextra-1:20190328-1.noarch

I have notice that texlive-rsc is only in present for i586 repo, when I install it :

package texlive-scheme-full-20190228-1.noarch requires texlive-collection-langjapanese, but none of the providers can be installed
 - package lyx-2.3.2-1.x86_64 requires texlive-scheme-full, but none of the providers can be installed
 - package texlive-collection-langjapanese-20190228-1.noarch requires texlive-japanese-otf-uptex, but none of the providers can be installed
 - conflicting requests
 - nothing provides perl(script::MakeSPList) needed by texlive-japanese-otf-uptex-1:0.18-2.noarch

I stopped here because I do not know of what is "perl(script::MakeSPList)"....

In conclusion, except for libhunspell the problem seems to be coming mainly from texlive...

## Comment 19707
Date: 2019-07-27 21:21:14 +0000
From: AngryPenguin &lt;<angrypenguinpoland@gmail.com>&gt;

Should be fixed now.

Some texlive package updated/fixed. Also new lyx backported from cooker to rolling and 4.0.
Packages in testing repo. To test it enable main/testing repository and try install lyx.

## Comment 19709
Date: 2019-07-29 22:46:56 +0000
From: @benbullard79

Moving these packages to rolling/release and 4.0/updates repositories.
